index.html

```diff
@@ -16,14 +16,15 @@
             <div class="logo">
                 <a href="#">My Logo</a>
             </div>
-            <div class="navbarMenue">
-                <ul>
-                    <li class="transition"><a href="#">Portfolio</a></li>
-                    <li class="transition"><a href="#">About</a></li>
-                    <li class="transition"><a href="#">Contact</a></li>
+            <div class="navbarMenue" id="navbarMenue">
+                <div class="close" id="close">x</div>
+                <ul class="navbarLinks">
```

Comment:
- [ ] The hamburger and closed button must follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). how about using a closed menu class icon, for example: `<i class="fa fa-close"></i>` instead of using a `div` tag with an `x` ? for more documentation you can check this [link](https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-close) or 🤔  you can download the icon image from template Figma design to make sure you are using the same icon with the original design 💯 👍🏻